examples/run_examples.sh

```diff
@@ -64,10 +69,34 @@ done
 MPLCONFIGDIR="$(pwd)/matplotlib/"
 export MPLCONFIGDIR
 
-time_format="  time: {real: %E, user: %U, system: %S}\n\
-  memory: {total: %K, max_rss: %M}"
+INFO_OS="$(uname -s)"
 
-basedir=$PWD
+# find "time" command
+if test "${INFO_OS:-}" = "Darwin"; then
+    if test -x "/usr/local/bin/gtime"; then
```

Comment:
For some reason, I cannot make a suggestion here, but the path should be `/opt/homebrew/bin/gtime` (also below). Homebrew installs everything under `/opt/homebrew` on Apple Silicon.

Reply:
A nicer way to define this would be a `$(which gtime)` but I don't know if that is correctly found. Could you check?

Reply:
Seems to work:
```
(nn) ~> echo $(which gtime)
/opt/homebrew/bin/gtime
```

Reply:
Thanks, I added the `which`